modules/tinymce/src/core/test/ts/browser/EditorTest.ts

```diff
@@ -469,6 +469,18 @@ describe('browser.tinymce.core.EditorTest', () => {
     input.parentNode.removeChild(input);
   });
 
+  it('TINY-6946: Images should be properly cleaned up and not cause a crash', () => {
```

Comment:
Maybe change this from crash to `Images should be properly cleaned up if they contain invalid trailing data` or something?